.github/workflows/archery.yml

```diff
@@ -44,9 +44,8 @@ jobs:
         with:
           fetch-depth: 0
       - name: Git Fixup
-        if: ${{ github.ref != 'master' }}
```

Comment:
Evidently it doesn't work on the master branch, so omit errors from the bash command instead.